src/main/resources/META-INF/accesstransformer.cfg

```diff
@@ -316,3 +316,6 @@ public net.minecraft.client.data.models.ItemModelGenerators$TrimMaterialData
 public net.minecraft.client.data.models.ItemModelGenerators$TrimMaterialData <init>(Ljava/lang/String;Lnet/minecraft/resources/ResourceKey;Ljava/util/Map;)V
 public-f net.minecraft.client.data.models.ModelProvider getName()Ljava/lang/String;
 public net.minecraft.client.data.models.ModelProvider$ItemInfoCollector register(Lnet/minecraft/world/item/Item;Lnet/minecraft/client/renderer/item/ClientItem;)V
+
+# Required to support sorting the client reload listeners.
+public-f net.minecraft.server.packs.resources.ReloadableResourceManager listeners
```

Comment:
Non-final public fields can be modified at any point, we should instead patch an internal-annotated method onto `ReloadableResourceManager` that receives a `SortedReloadListenerEvent` and replaces its map.

Reply:
Yeah good point, it would be hard for someone to spoof an entire `SortedReloadListenerEvent` to call the method.